### PR TITLE
[15.0][MIG] agreement_rebate_partner_company_group: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT39022

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @carlosdauden @sergio-teruel

**Review by carlosdauden — APPROVED** [review]

File: agreement_rebate_partner_company_group/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "version": "15.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Contract",
+    "website": "https://github.com/OCA/contract",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["agreement_rebate", "base_partner_company_group"],
+    "data": [],
```
Comment (sergio-teruel) [inline agreement_rebate_partner_company_group/__manifest__.py]: remove this empty key

**Review by sergio-teruel — APPROVED** [review]
Little comment but OK

File: agreement_rebate_partner_company_group/__manifest__.py (hunk 1 above)
Comment (ernestotejeda) [reply agreement_rebate_partner_company_group/__manifest__.py]: done

**Pull request merged**